### PR TITLE
Copy edits to bare metal docs

### PR DESCRIPTION
**Pull request by johnswil**
Hi, thank you for your contribution to Mantl! Before we can accept any code into
master, we need it to meet the following criteria. If there are any you can't
satisfy yourself, go ahead and open the pull request anyway and we'll help you
test. Feel free to delete this message once you're done. Thanks again!

- [ ] Installs cleanly on a fresh build of most recent master branch
- [ ] Upgrades cleanly from the most recent release
- [ ] Updates documentation relevant to the changes

### CODE REVIEW

File: docs/getting_started/bare-metal.rst [hunk 1]
```diff
@@ -52,16 +52,16 @@ During installation you will use the defaults except:
...
+      Try to leave at least 50 unformated for the Docker LVM partion
       that is described in the "Create Partion for Docker LVM" section
       below.
 
 -  Turn on your wired internet connection. It should just be a toggle
-   switch for your device.
--  Once the install starts it asks for a root password and a first user.
--  Having a ``centos`` admin user will match what happens in cloud
-   environments.
+   switch for your device
```
Comment (siddharthist) [inline docs/getting_started/bare-metal.rst]: Why was this period removed?

File: docs/getting_started/bare-metal.rst [hunk 2]
```diff
@@ -153,7 +153,7 @@ Create Partion For Docker LVM
 -  please note which partition it is in. So if its partition 5,
    eventually you will need to tell mantl /dev/sda5 for the LVM
 -  you kinda want all your machines to use the same partition because
-   this partition is entered as a system wide variable.
+   this partition is entered as a system wide variable
```
Comment (siddharthist) [inline docs/getting_started/bare-metal.rst]: Why was this period removed?

Comment (sehqlr) [inline docs/getting_started/bare-metal.rst]: If you look at all of the other items in the list, they don't end with punctuation. Removing the period makes it consistent. (it's a copyediting thing.)

File: docs/getting_started/bare-metal.rst (hunk 1 above)
Comment (sehqlr) [inline docs/getting_started/bare-metal.rst]: See my other note re: list style consistency

**Pull request merged**